.github/workflows/release-python.yml

```diff
@@ -60,6 +60,9 @@ jobs:
       - name: Add bigidx feature
         if: matrix.package == 'polars-u64-idx'
         run: tomlq -i -t '.dependencies.polars.features += ["bigidx"]' py-polars/Cargo.toml
+      - name: Update optional dependencies
+        if: matrix.package != 'polars'
+        run: sed $SED_INPLACE 's/polars\[/${{ matrix.package }}\[/' py-polars/pyproject.toml
```

Comment:
`SED_INPLACE` is not defined here. You should move the definition from the wheels job to the top-level `env` definition.

Reply:
@stinodego thanks for the review, really appreciate it! I now pushed a commit on top of this one that fixes the issue you raised. I also added one more change to wheel building, as it was missing the exact same change.